docs/configuration.md

```diff
@@ -1758,7 +1766,8 @@ Apart from these, the following properties are also available, and may be useful
     Off-heap buffers are used to reduce garbage collection during shuffle and cache
     block transfer. For environments where off-heap memory is tightly limited, users may wish to
     turn this off to force all allocations to be on-heap.
-    </td>
+  </td>
+  <td></td>
```

Comment:
@beliefer, can we add the version by tracking `common/network-common/src/main/java/org/apache/spark/network/util/TransportConf.java`?